src/client/components/measures-list/measures-list.vue

```diff
@@ -6,16 +6,40 @@
         :key="measureGroup.title"
         class="measures-list__group">
         <h2 class="measures-list__group-title">{{ measureGroup.title }}</h2>
-        <div
+        <md-card
           v-for="(measure, index) in measureGroup.items"
-          :key="measure.title">
-          <button
-            class="measures-list__select"
-            @click="selectMeasure({groupIndex, index})">
-            <strong>{{ measure.title }}</strong>
-            <span class="measures-list__select-caption">KIES</span>
-          </button>
-        </div>
+          :key="measure.title"
+          :class="{ 'measures-list__measure--active': selectedMeasure.id === measure.id }"
+          class="measures-list__measure"
+        >
+          <div class="measures-list__measure-header" >
+            <strong class="measures-list__measure-title">{{ measure.title }}</strong>
+            <md-button
+              class="measures-list__measure-select"
+              @click="selectMeasure({groupIndex, index})"
+            >
+              {{ selectedMeasure.id === measure.id ? 'Klaar' : 'Kies percelen' }}
+            </md-button>
+          </div>
+          <div class="measures-list__selected-parcels">
+            <template v-if="parcelsPerMeasure[measure.id]">
+              <div
```

Comment:
I'd reconsider using [Vue Material chips](https://vuematerial.io/components/chips) instead. They have all interactive states (hover, focus, ..., etc) and right touch areas and accessibility built-in.